src/NServiceBus.AcceptanceTests/Routing/When_handler_exists_but_routing_information_is_missing.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace NServiceBus.AcceptanceTests.Routing
+{
+    using System.Threading.Tasks;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NServiceBus.AcceptanceTests.ScenarioDescriptors;
+    using NUnit.Framework;
+
+    public class When_handler_exists_but_routing_information_is_missing : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_not_auto_subscribe_on_unicast_transports()
+        {
+            await Scenario.Define<Context>()
+                .WithEndpoint<Subscriber>()
+                .Done(c => c.EndpointsStarted)
+                .Repeat(r => r.For<AllTransportsWithMessageDrivenPubSub>())
+                .Should(c =>
+                {
+                    Assert.True(c.EndpointsStarted);
+                })
+                .Run();
+        }
+
+        public class Context : ScenarioContext
+        {
+        }
+
+        public class Subscriber : EndpointConfigurationBuilder
+        {
+            public Subscriber()
+            {
+                EndpointSetup<DefaultServer>();
+            }
+
+            public class MyEventHandler : IHandleMessages<MyEvent>
+            {
+                public Task Handle(MyEvent @event, IMessageHandlerContext context)
+                {
+                    return Task.FromResult(0);
+                }
+            }
+        }
+
+        public class MyEvent : IEvent
+        {
+        }
+    }
+}
```

Comment:
I assume the test failed before because the endpoint couldn't start up at all? The test isn't too easy to understand at first glance since you have to know that autosubscribe will throw an exception in case the information is missing.

can we rename the test + assertion or at least add a comment?

maybe `when_using_autosubscribe_with_missing_routing_information` `Should_not_throw_on_endpoint_startup`?